.gitignore

```diff
@@ -12,4 +12,4 @@ node_modules
 dist
 
 # Config
-src/config.ts
+config.ts
```

Comment:
Why moving my config file...

Reply:
think it was a mistake, I think I changed it back it later version